### PR TITLE
[develop] Enable run_post to work with varying forecast length hours

### PR DESCRIPTION
**Pull request by chan-hoo**
## DESCRIPTION OF CHANGES: 
- Build UPP for AQM (Online-CMAQ).
- Add four new cycles to the workflow XML file for real-time run with varying forecast length hours.
- Update the UPP input namelist and control file.

### Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## TESTS CONDUCTED: 
1. WE2E tests:
- community_ensemble_2mems_stoch  
- grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16  
- MET_verification

2. Sample script test for AQM
- config.aqm.community.yaml on Hera and WCOSS2
- config.aqm.nco.realtime.yaml on WCOSS2

- [x] hera.intel
- [ ] orion.intel
- [ ] cheyenne.intel
- [ ] cheyenne.gnu
- [ ] gaea.intel
- [ ] jet.intel
- [x] wcoss2.intel
- [ ] NOAA Cloud (indicate which platform)
- [ ] Jenkins
- [ ] fundamental test suite
- [ ] comprehensive tests (specify *which* if a subset was used)

## ISSUE: 
Fixes issue mentioned in https://github.com/NOAA-EMC/AQM/issues/63

## CHECKLIST
- [x] My code follows the style guidelines in the Contributor's Guide
- [x] I have performed a self-review of my own code using the Code Reviewer's Guide
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes need updates to the documentation. I have made corresponding changes to the documentation
- [ ] My changes do not require updates to the documentation (explain).
- [x] My changes generate no new warnings
- [x] New and existing tests pass with my changes
- [ ] Any dependent changes have been merged and published

### CODE REVIEW

File: modulefiles/build_hera_intel.lua
Comment (MichaelLueken) [inline modulefiles/build_hera_intel.lua]: I have reached out to Natalie and she has corrected the issue with netcdf/4.9.1 and hdf5/1.14.0 being treated as the defaults on Hera.  Please remove these two lines and make sure that the online-cmaq builds without issue.  Thanks!

**Review by MichaelLueken — COMMENTED** [review]
@chan-hoo Overall, these changes look good to me!  I have reached out to @natalie-perlin and she changed the defaults for `netcdf` and `hdf5` to `4.7.4` and `1.10.6`, rather than `4.9.1` and `1.14.0`, on Hera.

Please remove the additional lines in `modulefiles/build_hera_intel.lua` and make sure that the AQM builds and runs.  Once complete, I'll approve the change.  Thanks!

File: modulefiles/build_hera_intel.lua
Comment (chan-hoo) [reply modulefiles/build_hera_intel.lua]: @MichaelLueken, yes, I had an issue with netcdf on hera. Thank you for the update. I'll remove the lines and test it again.

Comment (chan-hoo) [reply modulefiles/build_hera_intel.lua]: Removed. It works!

**Review by MichaelLueken — APPROVED** [review]
@chan-hoo Thank you for removing the modification from `modulefiles/build_hera_intel.lua`!  I will go ahead and approve the changes now.  Please note that due to issue #635, I will hold off on submitting the Jenkins tests (also to allow for additional feedback from other reviewers).

**Review by danielabdi-noaa — APPROVED** [review]
Looks good to me

**Comment by chan-hoo** [comment]
Thanks, @danielabdi-noaa

**Comment by MichaelLueken** [comment]
@chan-hoo Please note that the Jenkins automated tests failed for Orion (same error as noted in issue #635).  This is due to the old version of Git that is used by default on the machine.  I'm manually running the fundamental WE2E tests on Orion now.  Once they pass (or fail), I'll add that information to this PR.

**Comment by chan-hoo** [comment]
@MichaelLueken, thank you!

**Comment by MichaelLueken** [comment]
@chan-hoo The WE2E fundamental tests were manually ran on Orion and all five tests successfully passed.  Once the rest of the Jenkins tests complete, this work will be merged to develop.

**Comment by MichaelLueken** [comment]
All tests successfully completed (with the exception of the tests on Orion).  Since manual tests on Orion passed, I will now merge this PR into develop.

**Pull request merged**
